packages/chrome/src/index.tsx

```diff
@@ -96,7 +114,7 @@ const App = () => {
     setColorMode,
   }
 
-  if (!context.theme) return false
+  if (!context.theme) return null
```

Comment:
Replaced line [156](https://github.com/system-ui/theme-ui/pull/909/files#diff-715666d45aadf8a39f99bb87b894facdR156) with `context.theme` since we it's the same value as `state.theme` and we have a guarantee that `theme` is defined and/or not null.